mtg_card_identifier/RootView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  RootView.swift
+//  mtg_card_identifier
+//
+//  Created by Dylan Martin on 8/11/23.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+struct RootViewReducer: Reducer {
+    struct State: Equatable {
+        var isDatabaseDownloaded = SQLiteFileManager.checkDatabaseStatus()
+        var downloadDatabaseState = DownloadDatabaseFilesViewReducer.State()
+    }
+    
+    enum Action: BindableAction {
+        case binding(BindingAction<State>)
+        case checkDatabaseDownloaded
+        case downloadDatabaseFilesViewAction(DownloadDatabaseFilesViewReducer.Action)
+    }
+    
+    var body: some Reducer<State, Action> {
+        BindingReducer()
+        
```

Comment:
You'll need to add a Scope(...) reducer for your sub-state/reducer (the DownloadDatabaseFilesViewReducer).